Cesium3DTilesContent/src/BinaryToGltfConverter.cpp

```diff
@@ -36,7 +36,26 @@ CesiumAsync::Future<GltfConverterResult> BinaryToGltfConverter::convert(
     const std::span<const std::byte>& gltfBinary,
     const CesiumGltfReader::GltfReaderOptions& options,
     const AssetFetcher& assetFetcher) {
-  return assetFetcher.asyncSystem.createResolvedFuture(
-      convertImmediate(gltfBinary, options, assetFetcher));
+  return _gltfReader
+      .readGltfAndExternalData(
+          gltfBinary,
+          assetFetcher.asyncSystem,
+          assetFetcher.requestHeaders,
+          assetFetcher.pAssetAccessor,
+          assetFetcher.baseUrl,
+          options)
+      .thenInWorkerThread(
+          [&assetFetcher](CesiumGltfReader::GltfReaderResult&& gltfResult) {
+            if (gltfResult.model) {
```

Comment:
Looks like the `convertImmediate` function above has been inlined here, so it can be removed.